packages/@blockprotocol/type-system/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blockprotocol/type-system",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
We don't need to bump version — this is done in a PR created by Changesets.

Reply:
Gotcha, cheers
```suggestion
  "version": "0.0.1",
```